MAUI/Themes/HowTo.md

```diff
@@ -20,18 +20,18 @@ void UpdateTheme(object sender, System.EventArgs e)
         var theme = mergedDictionaries.OfType<SyncfusionThemeResourceDictionary>().FirstOrDefault();
         if (theme != null)
         {
-            if (theme.SfVisualTheme is SfVisuals.MaterialDark)
+            if (theme.VisualTheme is SfVisuals.MaterialDark)
```

Comment:
Create the Hotfix PR also for UG changes

Reply:
Ok sir.